File: LIR.DOMAIN/Entities/ConsumerProfile.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LIR.DOMAIN.Entities
{
    public class ConsumerProfile : BaseModel
    {
        public string ConsumerName { get; set; }
        public double BasicSalary { get; set; }
        public DateTime Birthdate { get; set; }

        public ICollection<ConsumerBenefitResult> ConsumerBenefitResults { get; set; }
    }
}

File: LIR.WEB/angularUI/src/app/models/ConsumerProfile.ts
import { ConsumerBenefitResult } from "./ConsumerBenefitResult";

export class ConsumerProfile{
    consumerName: string = "";
    basicSalary: number = 0;
    birthdate: Date = new Date();
    consumerBenefitResults: ConsumerBenefitResult[] = [];
}